src/Chapter2.hs

```diff
@@ -620,7 +627,11 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+
+takeEven :: [a] -> [a]
+takeEven (x:_:xs) = x : takeEven xs
+takeEven (x:_) = [x]
+takeEven x = x
```

Comment:
I would recommend writing these two patterns explicitly to utilize GHC exhaustiveness checker for patterns. The first line is changed for minor aesthetics reasons, but the last one is the important change here.
```suggestion
takeEven [x] = [x]
takeEven [] = []
```